pkg/roachpb/BUILD.bazel

```diff
@@ -3,13 +3,13 @@ load("@io_bazel_rules_go//proto:def.bzl", "go_proto_library")
 load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
 load("//build:STRINGER.bzl", "stringer")
 
+# keep
```

Comment:
Ah, I don't like this. This will result in Gazelle never updating this target. Why did you make this change?